modules/@apostrophecms/job/index.js

```diff
@@ -118,7 +118,9 @@ module.exports = {
             // It's only relevant to pass a job ID to the notification if
             // the notification will show progress. Without a total number we
             // can't show progress.
-            jobId: total && job._id
+            jobId: total && job._id,
+            ids,
```

Comment:
There could be thousands of ids yeah? Pushing them all in every "in progress" notification could be a serious performance issue. How about pushing just those not previously sent in a notification?

Reply:
Or how about pushing the ids only in the final notification? That could be a reasonable compromise.

All of this is making me wish batch operations still used a modal. Then it would be very predictable UX: you can't do anything (at least in this tab) until the batch operation is done at which point you can expect some things to refresh. How great an idea is it really to try to work while a batch operation is making changes all over your database?

Reply:
I understand I'm probably not going to get my way on that last suggestion at least for beta, but we need to figure out how not to bang bang bang on the refresh of the piece manager.

Reply:
There's only one progress notification per job. Once there the notification takes over  polling the job progress. This is used for the `content-changed` event.